aiosparql/client.py

```diff
@@ -182,6 +182,8 @@ def _pretty_print_query(self, query: str) -> str:
                     exc.request_info, exc.history,
                     code=exc.code, message=exc.message, headers=exc.headers,
                     explanation=explanation)
+            if resp.content_type == 'text/html':
+                return resp
             return await resp.json()
```

Comment:
I don't think the return type matches. resp is probably an object response and the return type expected is a `dict`.

Reply:
Good point! I say your commit in add jena support and that's much better